src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -267,17 +294,50 @@ private string ClearHexString(string hexString)
             }
         }
 
+        private bool IsHex(string str) => str.Length % 2 == 0 && str.All(c => (c >= '0' && c <= '9') || (c >= 'a' && c <= 'f') || (c >= 'A' && c <= 'F'));
+
         /// <summary>
-        /// Converts an address to its corresponding scripthash
+        /// Fix for Base64 strings containing unicode
+        /// input:  DCECbzTesnBofh/Xng1SofChKkBC7jhVmLxCN1vk\u002B49xa2pBVuezJw==
+        /// output: DCECbzTesnBofh/Xng1SofChKkBC7jhVmLxCN1vk+49xa2pBVuezJw==
         /// </summary>
-        /// <param name="address">
-        /// String that represents the address to be converted
-        /// </param>
-        /// <returns>
-        /// Returns null when the string does not represent an address or when
-        /// it is not possible to parse the address to scripthash; otherwise returns
-        /// the string that represents the converted scripthash
-        /// </returns>
+        /// <param name="str">Base64 strings containing unicode</param>
+        /// <returns>Correct Base64 string</returns>
+        public string Base64Fixed(string str)
```

Comment:
I believe `Console.OutputEncoding = Utility.StrictUTF8;` will fix this problem.

Reply:
The purpose of this one is to handle the output of other wallets or SDKs that have Base64 with Unicode escaped characters in them